### PR TITLE
Feature/fix filter

### PR DESCRIPTION
**Pull request by ykobuk**
filters by drop-down and radio button

### CODE REVIEW

File: client/src/components/MapLayout/index.jsx [hunk 1]
```diff
@@ -1,80 +1,111 @@
...
-
-  renderAddEventButton() {
-    if (isLogged()) {
-      return (
-        <div className="fixed-action-btn horizontal">
-          <Link className="btn-floating btn-large red" to="/add-event">
-            <Icon>add</Icon>
-          </Link>
-        </div>
-      )
```
Comment (Merteg) [inline client/src/components/MapLayout/index.jsx:52]: Turn it back, please. Why did you delete this?

File: client/src/components/MapLayout/index.jsx [hunk 2]
```diff
@@ -1,80 +1,111 @@
...
       }
-      request('/events-short-info', 'POST', JSON.stringify(filterData)).then(data => {
-          if ('info' in data) {
-              this.setState({ events: data.info })
-          }
-      });
+    });
   }
 
   componentDidMount() {
```
Comment (Merteg) [inline client/src/components/MapLayout/index.jsx]: You have the same parts of code here and in filter method. You can just call a filter method here, I think.

File: client/src/components/MapLayout/index.jsx [hunk 3]
```diff
@@ -1,80 +1,111 @@
...
-              <option value='7'>One week</option>
-              <option value='14'>Two weeks</option>
-          </Input>
+        <Col>
+          <p>
+              <input
+                     className="with-gap"
+                     name="group1"
+                     type="radio"
+                     id="test1"
```
Comment (Merteg) [inline client/src/components/MapLayout/index.jsx]: :) In my opinion, `test` is not a very suitable id.

File: client/src/components/MapLayout/index.jsx [hunk 4]
```diff
@@ -1,80 +1,111 @@
...
         <Col s={4}>
           <div>
             &nbsp;
           </div>
         </Col>
         <Col s={8} className="map-page-container">
-          <EventMap events={this.state.events}/>
+          <EventMap events={this.state.events} />
         </Col>
-        {this.renderAddEventButton()}
```
Comment (Merteg) [inline client/src/components/MapLayout/index.jsx]: Also, this must be here.

File: server/server/views/category.py [hunk 5]
```diff
@@ -17,7 +17,7 @@ def __init__(self, request, context=None):
     def __acl__(self):
         return [(Allow, Authenticated, 'get')]
 
-    @view(permission='get')
+    #@view(permission='get')
```
Comment (Merteg) [inline server/server/views/category.py]: Remove...

**Review by Merteg — CHANGES_REQUESTED** [review]

File: client/src/components/MapLayout/index.jsx (hunk 3 above)
Comment (ykobuk) [reply client/src/components/MapLayout/index.jsx]: How about id = "radio_button_id_1"?)

Comment (Merteg) [reply client/src/components/MapLayout/index.jsx]: maybe `radio_1` will be enough. There is no reason to type 'id' in `id` property

Comment (ykobuk) [reply client/src/components/MapLayout/index.jsx]: May I write radio1 or I must to write underscore between radio and 1?

Comment (Merteg) [reply client/src/components/MapLayout/index.jsx]: It can be written together, but between different words, you must add an underscore

**Comment by ykobuk** [comment]
deleted some rows and called function filter in componentDidMount

File: client/src/components/MapLayout/index.jsx [hunk 6]
```diff
@@ -9,70 +9,100 @@ import SelectCategory from '../EventForm/SelectCategory';
...
-          </Input>
-          <Button onClick={this.filter}>Filter</Button>
-          <Col s={3}>
-              <SelectCategory onChangeHandler={this.onChangeHandler}/>
-          </Col>
+        <Col>
+          <p>
+            <input
+              className="with-gap"
+              name="group1"
```
Comment (RayMet) [inline client/src/components/MapLayout/index.jsx:58]: I don't like numbers in names, I think better call like group_one or group_one_day, and same with id names.

**Review by RayMet — CHANGES_REQUESTED** [review]

File: client/src/components/MapLayout/index.jsx (hunk 6 above)
Comment (Merteg) [reply client/src/components/MapLayout/index.jsx:58]: Can you argue?

Comment (ykobuk) [reply client/src/components/MapLayout/index.jsx:58]: Did you notice that every name='group1' in every input? This is for grouping the radio button, and if i write name="VasiaPypkin" the result will be same! It's just a string that should be the same for all "name" for radio button, it is not even a variable

**Review by RayMet — APPROVED** [review]

**Review by Merteg — APPROVED** [review]
